scripts/deploy_schema.py

```
@@ -0,0 +1,269 @@
+        cursor.execute("DELETE FROM rivet_users WHERE telegram_id = 999999999")
+        print_colored("✅ Test data cleaned up", GREEN)
+
+        cursor.close()
+        conn.close()
+
+        return True
+
+    except psycopg2.Error as e:
+        print_colored(f"\n❌ Database error: {e}", RED)
+        return False
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)